src/server/api/lib/twilio.js

```diff
@@ -450,6 +451,67 @@ async function handleIncomingMessage(message) {
   }
 }
 
+/**
+ * Create a new Twilio messaging service
+ */
+async function createMessagingService(friendlyName) {
```

Comment:
When you rebase can you try to match https://github.com/MoveOnOrg/Spoke/pull/1525/files#diff-5d117a0afdd02753cbd75a8890706ee4 to minimize merge conflicts between this branch and phone number management?

Reply:
Nevermind, I removed the messaging service functions from my PR to make it easier to merge with this. After rebasing, just copy over the getTwilio() function and change this to something like:

```js

async function createMessagingService(organization, friendlyName) {
  // TODO: document env var
  const twilioBaseUrl = getConfig("TWILIO_BASE_URL", organization);
  return await getTwilio(organization).messaging.services.create({
    friendlyName,
    statusCallback: urlJoin(twilioBaseUrl, "twilio-message-report"),
    // TODO: double check that appending orgId always works
    inboundRequestUrl: urlJoin(
      twilioBaseUrl,
      "twilio",
      organization.id.toString()
    )
  });
}
```